src/client/components/user_report_contents.tsx

```diff
@@ -155,10 +155,25 @@ export default function UserReportContents({ index, report, rootDomain }: UserRe
                         const searchParams = new URLSearchParams([
                           ["bug_file_loc_type", "allwordssubstr"],
                           ["bug_file_loc", rootDomain],
-                          ["component", "Site Reports"],
-                          ["product", "Web Compatibility"],
                           ["query_format", "advanced"],
                           ["resolution", "---"],
+                          ["j_top", "OR"],
+                          ["f1", "OP"],
+                          ["o2", "equals"],
+                          ["f2", "product"],
+                          ["v2", "Web Compatibility"],
+                          ["o3", "equals"],
+                          ["f3", "component"],
+                          ["v3", "Site Reports"],
+                          ["f4", "CP"],
+                          ["f5", "OP"],
+                          ["o6", "equals"],
+                          ["f6", "product"],
+                          ["v6", "Core"],
+                          ["o7", "equals"],
+                          ["f7", "component"],
+                          ["v7", "Privacy%3A Site Reports"],
```

Comment:
The `%3A` needs to be a plain `:` here. Otherwise, it would be double-encoded into a `%253A`.

Reply:
I looked at that thought "oh I better remember to unescape the code" and then… didn't. Sigh.